public/app/features/alerting/unified/components/contact-points/__mocks__/alertmanager.config.mock.json

```diff
@@ -100,7 +168,16 @@
       }
     ],
     "templates": ["slack-template", "custom-email", "provisioned-template", "template with spaces"],
-    "time_intervals": [],
+    "time_intervals": [
```

Comment:
(Just making these consistent with some of the k8s endpoints, because the mock server has validation in place to make sure the time intervals exist in the config, so with `alertingApiServer` disabled, the API call would fail)